docs/methods.md

```diff
@@ -1,155 +1,152 @@
-# Supplementary Bioinformatics Methods
+# `genome-sampler` installation and usage tutorial
 
-**NOTE: this is a work in progress**
+This document provides instructions for installing and using genome-sampler.
 
-## Environment setup
+## Installation instructions
 
-```
-conda env create -n az-covid-1 --file environment.yml
-```
+### Install Miniconda
```

Comment:
FWIW - it is possible to install via conda-package right now:
<img width="204" alt="Screen Shot 2020-06-05 at 5 36 17 PM" src="https://user-images.githubusercontent.com/274668/83931729-184c5000-a753-11ea-9e01-afcefeac7faa.png">

&

https://github.com/caporaso-lab/genome-sampler/blob/61367bf31a3db642bf3132b0b98d8c62614dce92/.github/workflows/main.yml#L55-L70

The problem is there is no good permalink for the build artifacts (yet) - that's what I'm working on right now. Just wanted to put this on your radar.

Reply:
The `fetch built package` step in the snippet above would translate to docs that look like "download the linux-64 or osx-64 ZIP file, then unzip into a directory called `built`. Then, you can just run the `conda install` command listed above" (although it would need some additional packages listed for things like snakemake et al).